pokemongo_bot/cell_workers/move_to_fort.py

```diff
@@ -109,7 +109,7 @@ def work(self):
                     formatted='Arrived at fort.'
                 )
 
-        return WorkerResult.SUCCESS
```

Comment:
There is no point to put your catch Pokemon task after the moving task. There is no point offering this kind of flexibility. People want the bot to work, they do not care about all the mighty features that they never asked for.
If you really want to remove the WorkerResult because you still want to allow people to put their moving tasks in the middle, why not just having a single variable that we use as a "lock" to decide who set the position ?

in MoveToFort
```
if self.bot.move_lock is not None and self.bot.move_lock != "MoveToFort":
    return

self.bot.move_lock == "MoveToFort"
<do my logic here>
```

in FollowPath
```
if self.bot.move_lock is not None and self.bot.move_lock != "FollowPath":
    return

self.bot.move_lock == "FollowPath"
<do my logic here>
```

That way you can put every move tasks where you want, and it only cost you 2 lines of code per walker.